### PR TITLE
DP-090 

### PR DESCRIPTION
**Pull request by greg-ray-lods**
# Module: 00
## Lab/Demo: 00

Fixes # .

Changes proposed in this pull request:

-
-
-

### CODE REVIEW

File: Instructions/Labs/01a-introduction-to-azure-databricks.md [hunk 1]
```diff
@@ -28,9 +28,9 @@ To use Azure Databricks, you first need to deploy an Azure Databricks workspace
 
 ### Create a cluster
 
-1. When your Azure Databricks workspace resource has been created, go to it in the portal, and select **Launch Workspace** to open your Databricks workspace in a new tab, signing in if prompted.
+1. When your Azure Databricks workspace resource has been created, select **Go to Resource** and select **Launch Workspace** to open your Databricks workspace in a new tab, signing in if prompted.
```
Comment (GraemeMalcolm) [inline Instructions/Labs/01a-introduction-to-azure-databricks.md:31]: No. Please leave step 1 as-is.

File: Instructions/Labs/01a-introduction-to-azure-databricks.md [hunk 2]
```diff
@@ -28,9 +28,9 @@ To use Azure Databricks, you first need to deploy an Azure Databricks workspace
 
 ### Create a cluster
 
-1. When your Azure Databricks workspace resource has been created, go to it in the portal, and select **Launch Workspace** to open your Databricks workspace in a new tab, signing in if prompted.
+1. When your Azure Databricks workspace resource has been created, select **Go to Resource** and select **Launch Workspace** to open your Databricks workspace in a new tab, signing in if prompted.
 
-1. In the left-hand menu of your Databricks workspace, select **Clusters**, and then select **+ Create Cluster** to add a new cluster with the following configuration:
+1. In the left-hand menu of your Databricks workspace, select **Create**, and then select **Cluster** to add a new cluster with the following configuration:
```
Comment (GraemeMalcolm) [inline Instructions/Labs/01a-introduction-to-azure-databricks.md:33]: No. Please change step 2 to:

1. In the left-hand menu of your Databricks workspace, select **Compute**, and then select **+ Create Cluster** to add a new cluster with the following configuration:
   - **Name**: *Enter a unique name.*
   - **Cluster Mode**: Single Node
   - **Pool**: None

File: Instructions/Labs/01a-introduction-to-azure-databricks.md [hunk 3]
```diff
@@ -45,7 +45,7 @@ To use Azure Databricks, you first need to deploy an Azure Databricks workspace
 
 ### Upload data
 
-1. Download `https://raw.githubusercontent.com/MicrosoftLearning/dp-090-databricks-ml/master/data/nyc-taxi.csv` to your computer, saving it as **nyc-taxi.csv** in any folder.
+1. Save `https://raw.githubusercontent.com/MicrosoftLearning/dp-090-databricks-ml/master/data/nyc-taxi.csv` to your computer by copying and pasting the entire page to notepad as **nyc-taxi.csv** in any folder.
```
Comment (GraemeMalcolm) [inline Instructions/Labs/01a-introduction-to-azure-databricks.md:48]: No. Please leave as-is

File: Instructions/Labs/01a-introduction-to-azure-databricks.md [hunk 4]
```diff
@@ -92,4 +92,4 @@ In this exercise, you will discover the Azure Databricks environment.
 
 ## Clean-up
 
-If you're finished working with Azure Databricks for now, in Azure Databricks workspace, on the **Clusters** page, select your cluster and select **&#9632; Terminate** to shut it down. Otherwise, leave it running for the next exercise.
+If you're finished working with Azure Databricks for now, in the left-hand menu of your Databricks workspace, select **Compute**, then select **■ Terminate** to shut it down. Otherwise, leave it running for the next exercise.
```
Comment (GraemeMalcolm) [inline Instructions/Labs/01a-introduction-to-azure-databricks.md:95]: No. Please leave as-is except change "the **Clusters** page" to "the **Compute** page.

**Review by GraemeMalcolm — CHANGES_REQUESTED** [review]
Please make the requested changes and resubmit the PR.

Thanks!

**Comment by greg-ray-lods** [comment]
I made the changes to the dp-090 fork. Are you seeing my pull request?

**Comment by GraemeMalcolm** [comment]
I merged this PR

**Comment by greg-ray-lods** [comment]
Ok so its complete?

**Comment by GraemeMalcolm** [comment]
Yes - I'll take it from here - thanks

**Pull request merged**